.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Miew CI/CD
+
+on: [push, pull_request]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18
+        
+    - name: Cache dependencies
+      uses: actions/cache@v3
+      with:
+        path: |
+          node_modules
+          ~/.sonar/cache
+          ~/.yarn
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    - name: Setup Code Climate test-reporter
```

Comment:
We don't need this entire step because we removed the CodeClimate reporter (perhaps, you wanted to remove only SonarCloud, but now Coveralls and CodeClimate are gone too).